app/views/questions/show.html.erb

```diff
@@ -9,15 +8,16 @@
 <% end %>
 
 <h5>Add comment:</h5>
-<%= form_for [@question, @comment] do |f|%>
-  <%= f.label :comment %>
-  <%= f.text_field :text %><br><br>
-  <%= f.submit 'Comment!' %>
-<% end %>
+  <%= render partial: "comments/question_comment", locals: {question: @question, comment: @comment} %>
+
 
 <h4>Answers:</h4>
 <% @question.answers.each do |ans| %>
   <%= render partial: "answers/show", locals: {answer: ans} %>
+    <% ans.comments.each do |c| %>
+      <p><%= c.user.username %> : <%= c.text %></p>
+    <% end %>
+    <%= render partial: "comments/answer_comment", locals: { answer: ans, comment: @comment } %>
```

Comment:
This would change to this:
<%= render partial: "comments/new", locals: {target: ans, comment: @comment} %>